components/CreateJournalEntry/index.tsx

```diff
@@ -0,0 +1,130 @@
+import React from "react";
+import Link from "next/link";
+import styles from "./create.module.scss";
+import { FaArrowLeft } from "react-icons/fa";
+import {BiImageAdd} from "react-icons/bi";
+import dynamic from "next/dynamic"
+import {useState} from "react";
+const ReactQuill = dynamic(import('react-quill'), { ssr: false});
+
+interface IFormValues {
+    title?: string,
+    description?: string,
+    image?: File,
+    body?: string,
+    category?: string, 
+    error?: boolean,
+    [key: string]: any, //TypeScript throws a weird error with this. This is set to any for now, but it can be changed later. 
```

Comment:
I believe this is the correct way to handle this, try to make sure you cast it to a known type when used